.github/workflows/ci.yml

```
@@ -26,24 +28,29 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install dependencies
+      - name: Install
         run: |
           python -m pip install --upgrade pip
           pip install --upgrade -r requirements-optional.txt
+          pip install --upgrade -e .
 
-      - name: Lint with black
-        run: |
-          pip install --upgrade black
-          black --check --diff green
+      - name: Format
+        run: black --check --diff green example
+        if: matrix.python-version == '3.11' && matrix.os == 'ubuntu-latest'
+
+      - name: Mypy
+        run: mypy .
         if: matrix.python-version == '3.11' && matrix.os == 'ubuntu-latest'
 
       - name: Test
-        run: python -m green.cmdline -tvvvv green
+        run: |
+          green -tvvvv green
+          cd example && python -m green.cmdline -tvvvv proj
```

If we're going to use a pip-installed version of green everywhere else, we might as well use it here too.

```suggestion
          cd example && green -tvvvv proj
```